control-plane-operator/controllers/hostedcontrolplane/kas/kms/azure.go

```diff
@@ -18,11 +18,11 @@ import (
 )
 
 const (
```

Comment:
Is this PR only making a naming cosmetic change?

Reply:
not only cosmetic change, aws consts were erroneously used at L129.
I renamed all the consts to start with `azure` prefix to avoid this happening again.